lib/diagnostic.js

```diff
@@ -38,19 +52,32 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = {
+  name: 'Dalek Blue',
+  species: 'Alien',
+  breed: 'Dalek',
+  vocalization: 'Exterminate!',
+  vocalize: function() {
+    // code goes in here
+  }
+};
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
-const Pet = function() {
-  /* Your answer here */
+const Pet = function(name, species, breed, vocalization) {
+  this.name: name;
+  this.species: species;
+  this.breed: breed;
+  this.vocalization: vocalization;
 };
 
 // 5. Attach a `vocalize` method to the Pet's prototype.
 
-/* your answer here */
+Pet.prototype.vocalize = function() {
+  // codes goes in here
+};
 
```

Comment:
Pet.prototype.vocalize = function(){
// stuff
   return this.vocalization;
 };